src/punx/github_handler.py

```diff
@@ -59,45 +59,45 @@
 
 def get_BasicAuth_credentials(creds_file_name = None):
```

Comment:
This function gets the GitHub credentials but they are no longer `BasicAuth` (username & password).  I suggest changing the name to `get_GitHub_credentials`.

Reply:
Request function name be changed.